tests/lib/tools.py

```diff
@@ -156,8 +156,6 @@ def setUpClass(cls):
             cls.assertIsNone = cls.emulate_assertIsNone
         if not hasattr(cls, "assertIsNotNone"):
             cls.assertIsNotNone = cls.emulate_assertIsNotNone
-        if hasattr(cls, "assertRaisesRegexp"):
```

Comment:
I guess this was needed for some old Python versions, but it is no longer needed, and it is producing a warning about assertRaisesRegexp being deprecated.